lib/rules/require-default-props.js

```diff
@@ -17,6 +17,9 @@ const report = require('../util/report');
 const messages = {
   noDefaultWithRequired: 'propType "{{name}}" is required and should not have a defaultProps declaration.',
   shouldHaveDefault: 'propType "{{name}}" is not required, but has no corresponding defaultProps declaration.',
+  noDefaultPropsWithFunction: 'Don\'t use defaultProps with function component.',
```

Comment:
```suggestion
  noDefaultPropsWithFunction: 'Don’t use defaultProps with function components.',
```
straight quotes are never correct in prose :-)